libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -229,24 +219,69 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
+            const contentComponentElement = this.contentComponent.getElementRef().nativeElement;
+
+            contentComponentElement.style.top = this.header.nativeElement.offsetHeight + 'px';
+            this._addClassNameToCustomElement(contentComponentElement, 'content-sticker');
         }
     }
 
+    /** @hidden */
+    private _listenOnScroll(): void {
+        if (this.contentComponent) {
+            const contentComponentElement = this.contentComponent.getElementRef().nativeElement;
+
+            this._subscriptions.add(
+                fromEvent(contentComponentElement, 'scroll')
+                    .pipe(debounceTime(20), throttleTime(20))
+                    .subscribe(() => {
+                        if (contentComponentElement.scrollTop > 0) {
+                            this._dynamicPageService.collapseHeader();
+                        } else {
+                            this._dynamicPageService.expandHeader();
+                        }
+                    })
+            );
+        }
+    }
+
+    /** @hidden Listen for window resize and adjust tab and content positions accordingly */
+    private _listenOnResize(): void {
+        this._subscriptions.add(
+            fromEvent(window, 'resize')
+                .pipe(debounceTime(60))
+                .subscribe(() => {
+                    console.log('calling resiize');
+                    this.setContainerPositions();
+                })
+        );
+    }
+
     /**
      * @hidden
      * set position for tabs and tabbed content's position relative to the tabs on scrolling
      */
     private _setTabsPosition(): void {
-        const tabList: HTMLElement = this._elementRef.nativeElement.querySelector('.fd-tabs');
-        if (tabList) {
-            tabList.style.top = this.header.nativeElement.offsetHeight + 'px';
-            tabList.style.position = 'fixed';
-            tabList.style.left = '0';
-            tabList.style.right = '0';
-            this.tabs.forEach((element) => {
-                element.contentTop = tabList.offsetHeight + 'px';
+        if (!this.contentContainer) {
+            return;
+        }
+        const tabList: HTMLElement = this.contentContainer.nativeElement.querySelector('.fd-tabs');
+        if (!tabList) {
+            return;
+        }
+
+        this._addClassNameToCustomElement(tabList, 'tab-sticker');
+        tabList.style.top = this.header.nativeElement.offsetHeight + 'px';
+
+        const tabContent = this.tabContents?.toArray();
+        if (tabContent) {
+            tabContent.forEach((contentItem) => {
+                const element: HTMLElement = contentItem
+                    .getElementRef()
+                    .nativeElement.querySelector('.fd-dynamic-page__content');
+                if (element) {
+                    element.style.top = this.header.nativeElement.offsetHeight + tabList.offsetHeight + 'px';
+                }
             });
         }
```

Comment:
That's also the thing that you don't need

Reply:
removed usage of top